charts/redpanda/values.schema.json

```diff
@@ -349,16 +349,142 @@
       "type": "object",
       "required": [
         "hostPath",
-        "persistentVolume"
+        "persistentVolume",
+        "tiered"
       ],
       "properties": {
         "hostPath": {
           "type": "string"
         },
+        "tiered": {
+          "type": "object",
+          "required": [
+            "mountType"
+          ],
+          "properties": {
+            "mountType": {
+              "type": "string",
+              "pattern": "^(none|hostPath|emptyDir|persistentVolume)$"
+            },
+            "hostPath": {
+              "type": "string"
+            },
+            "persistentVolume": {
+              "type": "object",
+              "required": [
+                "storageClass",
+                "labels",
+                "annotations"
+              ],
+              "properties": {
+                "storageClass": {
+                  "type": "string"
+                },
+                "labels": {
+                  "type": "object"
+                },
+                "annotations": {
+                  "type": "object"
+                }
+              }
+            },
+            "config":{
+              "type": "object",
+              "required": [
+                "cloud_storage_enabled",
+                "cloud_storage_region",
+                "cloud_storage_bucket"
+              ],
+              "properties": {
```

Comment:
Those fields are not guaranteed as per https://github.com/redpanda-data/helm-charts/pull/278#discussion_r1072782702